rising/transforms/affine.py

```diff
@@ -37,6 +37,7 @@ def __init__(self,
                  padding_mode: str = 'zeros',
                  align_corners: bool = False,
                  reverse_order: bool = False,
+                 per_sample: bool = True,
```

Comment:
just wondering, if we should let this default to false for now to be consistent to the other trafos and change this in the future.

Reply:
I was also thinking about that. I think we will need to decide on a per transform basis. 
E.g. in this case there is no drawback to have it on
With our current mirror implementation on the other we will need a loop to do this whil won't be the best performance? So I would leave it false there.
In general we will need to introduce automatic stacking of affine transforms und refactor some of our spatial transforms to affine ones to get max performance for the per sample case

Reply:
> So I would leave it false there.

You mean leave it true? Or change it to false?

Reply:
I mean leave it true 🙈